### ISSUE TITLE
 Please run 'zcashd-wallet-tool' even though I already have

### ISSUE THREAD

**Issue by PRabahy** [ZcashFoundation/zecwallet#376]
I'm still getting  `No accounts. Please run 'zcashd-wallet-tool' to create your wallet first...`, but I already ran that (twice). When I tried to run it the first time, it said I needed to add `-exportdir` to the command line. I put `-exportdir=C:\Users\USERNAME\AppData\Roaming\Zcash` and it created export20220730. I've tried running zcashd both with the `-exportdir` parameter and without it, and both give the same results.

Any ideas?

**Comment by adityapk00** [comment]
Please make sure that the tool finishes running and that you actually have some accounts. The tool will export the seed phrase and check that you have it. 

Did the tool finish successfully?

**Comment by PRabahy** [comment]
Yep. It finished running and had me verify 3 of the words.

**Comment by Novgorod** [comment]
Can confirm, it's a known bug. The new full node doesn't run on Windows anymore.

**Comment by tarvcode** [comment]
Been trying to get it to work for a few days on Windows.  Can't restore or upgrade an old wallet, can't create a new one.  Can't import keys.  Tried rolling back versions, no go.

**Comment by adityapk00** [comment]
Please remember that Zecwallet Fullnode is just a UI to the `zcashd` full node wallet. As such, it needs a fully synced `zcashd` node, with the wallet initialized to work. 

Is your full node synced? The requirements to sync have been getting higher, so this is a fairly resource-intensive process. Also, please note that zcashd is deprecated every 16 weeks, so you can't really go back to an old version. 

My only advice for folks having trouble in this thread is to use a light wallet like Zecwallet Lite. You can import your seed phrase or private key into the lightwallet and get access to your funds. 

Since The full node wallet depends on zcashd, there isn't much I can do to help.

**Comment by tarvcode** [comment]
Yes, my last step was having a fully sync'd zcashd running and then trying to use the wallet tool to generate the seed.  It seems to work but the GUI still says no wallets.

Tried letting it create a new wallet completely, but still gives the error.

I'm currently moving everything to a Debian environment see if I have better luck there.

Did try the lite wallet, but I'm trying to recover the funds in a shielded address.

**Comment by tarvcode** [comment]
Ok, at some point on Windows the zcashd decided to do a "Rescanning".  After it did this, I am able to access my wallet in the GUI and my private keys look to be correct.

Perhaps this is just an order of operations problem on the user end.

**Comment by SpankingFuzz** [comment]
> Ok, at some point on Windows the zcashd decided to do a "Rescanning". After it did this, I am able to access my wallet in the GUI and my private keys look to be correct.
> 
> Perhaps this is just an order of operations problem on the user end.

I installed v1.8.3 & got same error
It did a rescan & synced fully but I am still getting same error & I can see the DB folder & wallet.dat are being read about every 10 min
Any help is appreciated

**Comment by agowa338** [comment]
Can confirm this issue. I can see my balance within zcashd, but Zecwallet cannot find any account.
(It also fails to start the integrated zcashd itself, so maybe these issues are related?

**Comment by skironDotNet** [comment]
It's a bug but you can solve it by calling `zcash-cli z_getnewaccount` it will create an account in the wallet with no addresses but it's enough to make UI happy. 

Example:
```
skiron@machine:~/crypto/zec$ ./zcash-cli z_listaccounts
[
]

skiron@machine:~/crypto/zec$ ./zcash-cli z_getnewaccount
{
  "account": 0
}

skiron@machine:~/crypto/zec$ ./zcash-cli z_listaccounts
[
  {
    "account": 0,
    "addresses": [
    ]
  }
]

```


Now I don't know if this affects the seed (HD addresses generation) should not but just saying, zcash is so buggy you never know...

**Comment by skironDotNet** [comment]
@agowa338 to overcome "fails to start the integrated zcashd itself", please see
https://github.com/ZcashFoundation/zecwallet/issues/388#issuecomment-1414160492

**Comment by agowa338** [comment]
@skironDotNet I cannot confirm that your workaround is fully working. The UI may not show any error message now, BUT it also doesn't work. It only shows as "connected", however I cannot click on anything. It still doesn't show anything else than a blank page.
